diff --git a/node_modules/@openzeppelin/contracts/token/ERC1155/ERC1155.sol b/node_modules/@openzeppelin/contracts/token/ERC1155/ERC1155.sol
index 634f389..9bf3fb9 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC1155/ERC1155.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC1155/ERC1155.sol
@@ -20,7 +20,7 @@ contract ERC1155 is Context, ERC165, IERC1155, IERC1155MetadataURI {
     using Address for address;
 
     // Mapping from token ID to account balances
-    mapping(uint256 => mapping(address => uint256)) private _balances;
+    mapping(uint256 => mapping(address => uint256)) internal _balances;
 
     // Mapping from account to operator approvals
     mapping(address => mapping(address => bool)) private _operatorApprovals;
